### PR TITLE
respawn needs to have a value

### PR DESCRIPTION
**Pull request by simonschmeisser**
It's safer to stop instead of restarting and ignoring errors

### CODE REVIEW

File: launch/ur_common.launch [hunk 1]
```diff
@@ -26,7 +26,7 @@
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
 
   <!-- driver -->
-  <node name="ur_driver" pkg="ur_modern_driver" type="ur_driver" output="screen" respawn="">
+  <node name="ur_driver" pkg="ur_modern_driver" type="ur_driver" output="screen" respawn="false">
```
Comment (gavanderhoorn) [inline launch/ur_common.launch]: I would suggest to just remove the `respawn` attribute.

**Comment by simonschmeisser** [comment]
Now if I could squash commits using the web interface ...

**Comment by gavanderhoorn** [comment]
We can squash when we merge.

Seems `respawn` was added by @Zagitta, it's not present upstream.

**Comment by simonschmeisser** [comment]
I think it is a leftover from the script topic PR

**Review by gavanderhoorn — APPROVED** [review]
(unchecked) respawn on a driver is a rather bad idea.

**Comment by gavanderhoorn** [comment]
Thanks @simonschmeisser 👍

**Pull request merged**